plugins/module_utils/_auth_method_k8s.py

```diff
@@ -37,7 +38,8 @@ def validate(self):
                     self._options.set_option('kubernetes_token', token_file.read().strip())
 
         if self._options.get_option('kubernetes_token') is None:
-            raise HashiVaultValueError(self._options.get_option('kubernetes_token')+self._options.get_option_default('kubernetes_token_path')+"No Kubernetes Token specified or discovered.")
+            raise HashiVaultValueError(self._options.get_option_default('kubernetes_token_path') + 
+                                       " No Kubernetes Token specified or discovered.")
```

Comment:
Formatting preference, and also changed to `get_option` because this option should never be missing (it has a default value right?).

```suggestion
            raise HashiVaultValueError(
                self._options.get_option('kubernetes_token_path') + 
                " No Kubernetes Token specified or discovered."
            )
```

Reply:
yes the defaut path of kubernetes token in pods